### DISCUSSION TITLE
Upgrading NocoDB [Jul 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2618] [Discord Threads]
> 🙋‍♂️ **WM31**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@535455522435104798> refer details here https://docs.nocodb.com/getting-started/upgrading  
#  
> 🙋‍♂️ **WM31**  
#### So I can just swap the docker image to the new latest and start it, Nocodb will take up the existing MySQL DB just fine?  
#  
> 👨‍🦲 **dstala**  
#### just ensure environment variables configured are same   
#### https://docs.nocodb.com/getting-started/upgrading#pull-the-latest-nocodb-image-with-same-environment-variables  
#  
> 🙋‍♂️ **WM31**  
#### My NC_DB was stored in the docker-compose so it should be retained  
#  
> 🙋‍♂️ **WM31**  
#### The data volume set for /use/app/data for the Nocodb image should be retained as well?  
#  
> 🙋‍♂️ **wkw**  
#### > My NC_DB was stored in the docker-compose so it should be retained   
####    
#### i doubt that  
#  
> 🙋‍♂️ **wkw**  
#### > The data volume set for /use/app/data for the Nocodb image should be retained as well?   
####    
#### as long as u've mounted it - yes  
#  
> 🙋‍♂️ **WM31**  
#### The NC_DB environment variables is stored within the Nocodb definition, so I suppose everytime I start from down the same variable will be injected to the start up  
#  
> 🙋‍♂️ **wkw**  
#### As long as it s an external one then it will be fine  
#  
> 🙋‍♂️ **wkw**  
#### If it s pointing to somewhere inside docker (SQLite file) , n u hvnt mounted , then it ll be lost  
#  
> 🙋‍♂️ **WM31**  
#### Ok thanks for the tip  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/991913035163779103)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
